plugins/destination/duckdb/client/types.go

```diff
@@ -3,100 +3,95 @@ package client
 import (
 	"strings"
 
-	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/apache/arrow/go/v12/arrow"
+	"github.com/cloudquery/plugin-sdk/v2/types"
 )
 
-func (*Client) SchemaTypeToDuckDB(t schema.ValueType) string {
-	switch t {
-	case schema.TypeBool:
+func (c *Client) SchemaTypeToDuckDB(t arrow.DataType) string {
+	switch v := t.(type) {
+	case *arrow.ListType:
+		return c.SchemaTypeToDuckDB(v.Elem()) + "[]"
+	case *arrow.BooleanType:
 		return "boolean"
-	case schema.TypeInt:
+	case *arrow.Int8Type:
+		return "tinyint"
+	case *arrow.Int16Type:
+		return "smallint"
+	case *arrow.Int32Type:
+		return "int"
+	case *arrow.Int64Type:
 		return "bigint"
-	case schema.TypeFloat:
-		return "real"
-	case schema.TypeUUID:
-		return "uuid"
-	case schema.TypeString:
-		return "text"
-	case schema.TypeByteArray:
+	case *arrow.Uint8Type:
+		return "utinyint"
+	case *arrow.Uint16Type:
+		return "usmallint"
+	case *arrow.Uint32Type:
+		return "uint"
+	case *arrow.Uint64Type:
+		return "ubigint"
+	case *arrow.Float32Type:
+		return "float"
+	case *arrow.Float64Type:
+		return "double"
+	case *arrow.BinaryType:
+		return "blob"
+	case *arrow.LargeBinaryType:
 		return "blob"
-	case schema.TypeStringArray:
-		return "text[]"
-	case schema.TypeTimestamp:
+	case *types.UUIDType:
+		return "uuid"
+	case *types.JSONType:
+		return "json"
+	case *arrow.TimestampType:
 		return "timestamp"
-	case schema.TypeJSON:
-		return "text"
-	case schema.TypeUUIDArray:
-		return "uuid[]"
-	case schema.TypeCIDR:
-		return "text"
-	case schema.TypeCIDRArray:
-		return "text[]"
-	case schema.TypeMacAddr:
-		return "text"
-	case schema.TypeMacAddrArray:
-		return "text[]"
-	case schema.TypeInet:
-		return "text"
-	case schema.TypeInetArray:
-		return "text[]"
-	case schema.TypeIntArray:
-		return "int[]"
+	case *arrow.Date64Type:
```

Comment:
I wasn't able to find that it stores it in milliseconds in the code

Reply:
What do you mean? That's how the type is defined here https://arrow.apache.org/docs/python/generated/pyarrow.date64.html#pyarrow.date64 "milliseconds since UNIX epoch 1970-01-01"?

Reply:
I think it's a python thing but anyway changed it as prob there is no reason for `date64` anyway as `date32` should be enough